lib/qs.rb

```diff
@@ -28,10 +33,11 @@ def self.init
 
   def self.reset!
     self.config.reset
-    @serializer   = nil
-    @deserializer = nil
-    @client       = nil
-    @redis        = nil
+    @dispatcher_queue = nil
```

Comment:
@kellyredding - Qs will generically throw errors if its not initialized. 

I don't think this specifically cause any errors, unless something outside of Qs is going `Qs.dispatcher_queue.<something>`. The `client` being `nil` is the most likely thing to cause errors.